src/VisualStudio/IntegrationTest/TestUtilities/InProcess/Editor_InProc.cs

```diff
@@ -141,6 +140,16 @@ public void SetUseSuggestionMode(bool value)
             }
         }
 
+        public void VerifyNotSaved()
```

Comment:
Would naming these methods `Assert*` make it more clear that they will throw?

Reply:
Not sure. Either way seems to work.